modules/core/shared/src/main/scala/gem/parser/HorizonsDesignation.scala

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.parser
+
+import gem.target.HorizonsDesignation
+import gem.target.HorizonsDesignation._
+
+import atto._, Atto._
+import cats.implicits._
+
+/** Parser for [[gem.target.HorizonsDesignation]]. */
+trait HorizonsDesignationParsers {
+
+  private def des[A,B](s: String, p: Parser[A]): Parser[A] =
+    string(s"${s}_") ~> p
```

Comment:
Probably don't need type param `B` here.

Reply:
Oops, that's left over from an earlier version.